src/SDKs/Monitor/Management.Monitor/generate.cmd

```diff
@@ -5,4 +5,4 @@
 
 @echo off
 call %~dp0..\..\..\..\tools\generate.cmd monitor/resource-manager %*
-call %~dp0..\..\..\..\tools\generate.cmd monitor/data-plane %*
```

Comment:
This is one of the breaking changes: now we do not have data-plane. Actually we never had it, all the calls were going same endpoints and ding similar things.
We just moved the API's and models to the resource-manager area, i.e. one of the clients disappeared.

Also, the namespaces for the generated classes were fixed since they were incorrect before.